tests/Behat/Element/Admin/OrderCreateFormElement.php

```diff
@@ -262,12 +262,19 @@ private function clickOnTabAndWait(string $tabName): void
             ;
         });
     }
-
     private function waitForFormToLoad(): void
     {
         $form = $this->getDocument()->find('css', '[name="sylius_admin_order_creation_new_order"]');
         $this->getDocument()->waitFor(1000, function () use ($form) {
             return !$form->hasClass('loading');
         });
     }
+
+    public function isAddPaymentButtonVisible(): bool
+    {
+        return $this
+                ->getElement('payments')
+                ->find('css', '[data-form-collection="Add"]') !== null
```

Comment:
Formatting gone wrong here :( And I suppose it should be `'[data-form-collection="add"]'`?